### PR TITLE
Adding nuget package to Azure Artifacts

### PR DESCRIPTION
**Pull request by joebeernink**

### CODE REVIEW

File: .github/workflows/OntologyMapper_PR.yml [hunk 1]
```diff
@@ -1,13 +1,14 @@
-name: .NET
+name: Pull Request Build Only
```
Comment (Tyler-Angell) [inline .github/workflows/OntologyMapper_PR.yml:1]: Only?

File: .github/workflows/OntologyMapper_Prod.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+          source-url: ${{ env.AZURE_ARTIFACTS_FEED_URL }}
+        env:
+          NUGET_AUTH_TOKEN: ${{ secrets.AZURE_ARTIFACTS_PAT }} 
+
+      # Run dotnet build and package
+      - name: dotnet build and publish
```
Comment (Tyler-Angell) [inline .github/workflows/OntologyMapper_Prod.yml:48]: Why is this doing this again? the build from line 14 already did this?

File: OntologyMapper/src/Mappings/Mapped/Json/v0/BrickRec/mapped_json_v0_dtdlv2_Brick_1_3-REC_4_0.json [hunk 3]
```diff
@@ -5480,15 +5480,15 @@
     {
       "OutputDtmiFilter": "*",
       "OutputPropertyName": "externalIds",
-      "InputPropertyName": "mappingKey",
+      "InputPropertyNames": [ "mappingKey" ],
```
Comment (Tyler-Angell) [inline OntologyMapper/src/Mappings/Mapped/Json/v0/BrickRec/mapped_json_v0_dtdlv2_Brick_1_3-REC_4_0.json:5483]: These changes seem unrelated to Adding Nuget Packages to Azure Artifacts. Are they intended to be added?

File: .github/workflows/OntologyMapper_Prod.yml (hunk 2 above)
Comment (joebeernink) [reply .github/workflows/OntologyMapper_Prod.yml:48]: Fixing
